rosbag2_transport/src/rosbag2_transport/player.hpp

```diff
@@ -22,6 +22,14 @@
 #include <string>
 #include <unordered_map>
 
+// Terminal
+#if !defined(_MSC_VER)
+  #include <termios.h>
+  #include <unistd.h>
+#else
+  #include <windows.h>
+#endif
+
```

Comment:
as per previous comment, I think it makes sense to hide these `#if/#else` behind a new header/class for handling all specifics with the keyboard.

Reply:
fd97872